openedx/features/pakx/lms/overrides/tasks.py

```diff
@@ -127,7 +129,10 @@ def check_and_send_email_to_course_learners():
                 'language': get_user_preference(item.user, LANGUAGE_KEY),
                 'completed': grades.percent >= 1.0,
```

Comment:
we need to update this `completed` flag accordingly as per new logic.

Reply:
Done!